mne_bids/mne_bids.py

```diff
@@ -689,18 +700,19 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
                overwrite, verbose)
 
     # TODO: Implement coordystem.json and electrodes.tsv for EEG and  iEEG
-    if kind == 'meg':
+    if kind == 'meg' and not emptyroom:
         _coordsystem_json(raw, unit, orient, manufacturer, coordsystem_fname,
                           overwrite, verbose)
 
     events = _read_events(events_data, raw)
-    if len(events) > 0:
+    if len(events) > 0 and not emptyroom:
         _events_tsv(events, raw, events_fname, event_id, overwrite, verbose)
 
     make_dataset_description(output_path, name=" ", verbose=verbose)
     _sidecar_json(raw, task, manufacturer, sidecar_fname, kind, overwrite,
                   verbose)
-    _channels_tsv(raw, channels_fname, overwrite, verbose)
+    if not emptyroom:
```

Comment:
I don't see any harm in leaving in the channels files. It will probably make life easier also when reading them back in

Reply:
actually never mind, I think this is okay

Reply:
I just removed it because in the spec it only has the raw file and sidecar listed as files needed. But it can easily be kept in of course